Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -608,6 +613,66 @@ public void OnMouseEvent (MouseEventArgs a)
     /// <param name="ctrl">If <see langword="true"/> simulates the Ctrl key being pressed.</param>
     public abstract void SendKeys (char keyChar, ConsoleKey key, bool shift, bool alt, bool ctrl);
 
+    /// <summary>
+    ///     Provide handling for the terminal start reporting mouse events.
+    /// </summary>
+    public abstract void StartReportingMouseMoves ();
+
+    /// <summary>
+    ///     Provide handling for the terminal stop reporting mouse events.
+    /// </summary>
+    public abstract void StopReportingMouseMoves ();
+
+    /// <summary>
+    ///     Provide handling for the terminal write ANSI escape sequence request.
+    /// </summary>
+    /// <param name="ansiRequest">The <see cref="AnsiEscapeSequenceRequest"/> object.</param>
+    /// <returns>The request response.</returns>
+    public abstract string WriteAnsiRequest (AnsiEscapeSequenceRequest ansiRequest);
+
+    internal bool WriteAnsiRequestDefault (string ansi)
```

Comment:
Can we call this WriteRaw or something? it is a method on console driver that is purely writing out the string given to std out.  That could wind up being more generic than just ansi requests e.g. I Could use it in sixel to move the draw code up to the base class.

Reply:
I already removed that method. Now it's processed in each driver in the `WriteAnsiRequest` method, because they have different ways to handle the requests.